include/SDL3/SDL_gpu.h

```diff
@@ -1178,84 +1178,84 @@ typedef struct SDL_GPUTransferBufferCreateInfo
 
 typedef struct SDL_GPURasterizerState
 {
-    SDL_GPUFillMode fillMode;
-    SDL_GPUCullMode cullMode;
+    SDL_GPUFillMode fill_mode;
+    SDL_GPUCullMode cull_mode;
     SDL_GPUFrontFace frontFace;
-    SDL_bool depthBiasEnable;
+    SDL_bool enable_depth_bias;
     Uint8 padding1;
     Uint8 padding2;
     Uint8 padding3;
-    float depthBiasConstantFactor;
-    float depthBiasClamp;
-    float depthBiasSlopeFactor;
+    float depth_bias_constant_factor;
+    float depth_bias_clamp;
+    float depth_bias_slope_factor;
 } SDL_GPURasterizerState;
 
 typedef struct SDL_GPUMultisampleState
 {
-    SDL_GPUSampleCount sampleCount;
-    Uint32 sampleMask;
+    SDL_GPUSampleCount sample_count;
+    Uint32 sample_mask;
 } SDL_GPUMultisampleState;
 
 typedef struct SDL_GPUDepthStencilState
 {
-    SDL_bool depthTestEnable;
-    SDL_bool depthWriteEnable;
-    SDL_bool stencilTestEnable;
+    SDL_bool enable_depth_test;
+    SDL_bool enable_depth_write;
+    SDL_bool enable_stencil_test;
     Uint8 padding1;
-    SDL_GPUCompareOp compareOp;
-    SDL_GPUStencilOpState backStencilState;
-    SDL_GPUStencilOpState frontStencilState;
-    Uint8 compareMask;
-    Uint8 writeMask;
+    SDL_GPUCompareOp compare_op;
+    SDL_GPUStencilOpState back_stencil_state;
+    SDL_GPUStencilOpState front_stencil_state;
+    Uint8 compare_mask;
+    Uint8 write_mask;
     Uint8 padding2;
     Uint8 padding3;
 } SDL_GPUDepthStencilState;
 
 typedef struct SDL_GPUColorAttachmentDescription
 {
     SDL_GPUTextureFormat format;
-    SDL_GPUColorAttachmentBlendState blendState;
+    SDL_GPUColorAttachmentBlendState blend_state;
 } SDL_GPUColorAttachmentDescription;
 
 typedef struct SDL_GPUGraphicsPipelineAttachmentInfo
 {
-    const SDL_GPUColorAttachmentDescription *colorAttachmentDescriptions;
-    Uint32 colorAttachmentCount;
-    SDL_bool hasDepthStencilAttachment;
+    const SDL_GPUColorAttachmentDescription *color_attachment_descriptions;
```

Comment:
color_attachments?

Reply:
While we're at it I'd like to rename the "attachment" terminology to "target".

Reply:
Also I'm reluctant to drop "description" here because it is not a color target, the item being passed in is quite literally a description of a kind of color target.